app/models/order.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class Order < ApplicationRecord
+  enum status: { status_reject: 0, status_approved: 1, status_pending: 2 }, _default: :status_pending
```

Comment:
```suggestion
  enum status: { reject: 0, approved: 1, pending: 2 }, _default: :pending
```
Do tên trường em đặt là status rồi, nên giá trị em không cần thêm chữ status nữa đâu. Như vậy sẽ bị lặp.
Nếu e muốn lúc sử dụng các method enum có chữ status phía trước, thì có thể thêm options `_prefix: :status` phía cuối của line enum